lucene/CHANGES.txt

```diff
@@ -15,7 +15,7 @@ API Changes
 * LUCENE-8474: RAMDirectory and associated deprecated classes have been
   removed. (Dawid Weiss)
 
-* LUCENE-3041: The deprecated Weight#extractTerms() method has been 
+* LUCENE-3041: The deprecated Weight#extractTerms() method has been
```

Comment:
same issue like in previous PR!

Reply:
Well actually this PR predates other :), but yes same issue.